packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -106,6 +106,11 @@ export class VaCheckbox {
    */
   @Prop() messageAriaDescribedby?: string;
 
+  /**
```

Comment:
I added this prop because the va-statement-of-truth component has a va-checkbox that needs to have a specific name attribute